controllers/user/activate.js

```diff
@@ -24,6 +25,10 @@ router
     .route('/')
     .all(injectCopy('user.activate'))
     .get(async function(req, res) {
+        // Prevent activated users from persisting on this page
+        if (req.user && isActivated(req.user)) {
```

Comment:
Is this not the same as what `requireUnactivatedUser` does? Which is currently only on the POST endpoint, could we add that to `.all` instead?

Reply:
No – we used to have that here but removed it so that unauthed users could still activate (eg. when using another device): https://github.com/biglotteryfund/blf-alpha/pull/2581